critiquebrainz/frontend/external/bookbrainz_db/author.py

```diff
@@ -0,0 +1,110 @@
+import uuid
+from brainzutils import cache
+from typing import List
+import sqlalchemy
+import critiquebrainz.frontend.external.bookbrainz_db as db 
+from critiquebrainz.frontend.external.bookbrainz_db import DEFAULT_CACHE_EXPIRATION
+from critiquebrainz.frontend.external.bookbrainz_db.identifiers import fetch_bb_external_identifiers
+from critiquebrainz.frontend.external.bookbrainz_db.relationships import fetch_relationships
+
+def get_author_by_bbid(bbid: uuid.UUID) -> dict:
+	"""
+	Get info related to an author using its BookBrainz ID.
+	Args:
+		bbid : BBID of the author.
+	Returns:
+		A dictionary containing the basic information related to the author.
+		It includes the following keys:
+			- bbid: BookBrainz ID of the author.
+			- name: Name of the author.
+			- sort_name: Sort name of the author.
+			- author_type: Type of the author.
+			- disambiguation: Disambiguation of the author.
+			- identifiers: A list of dictionaries containing the basic information on the identifiers.
+			- rels: A list of dictionaries containing the basic information on the relationships.
+		
+		Returns None if the author is not found.
+	"""
+	author = fetch_multiple_authors([bbid])
+	if not author:
+		return None
+	return author[bbid]
+
+
+def fetch_multiple_authors(bbids: List[uuid.UUID]) -> dict:
```

Comment:
is there a reason why the literary works version of this method takes a limit and offset, but this one and series don't?

Reply:
Yes, I added the support for showing works on author pages in that PR itself. That is why it takes in limits and offset.